tsp-output/@azure-tools/typespec-csharp/src/Generated/Audio.cs

```diff
@@ -241,64 +238,66 @@ public virtual ClientResult<CreateTranslationResponse> CreateTranslation(CreateT
         /// <list type="bullet">
         /// <item>
         /// <description>
-        /// This <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md">protocol method</see> allows explicit creation of the request and processing of the response for advanced scenarios.
+        /// This <see href="https://azsdk/net/protocol/quickstart">protocol method</see> allows explicit creation of the request and processing of the response for advanced scenarios.
```

Comment:
is this link correct? I cannot open it.

Reply:
I uses Michael's alternative: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/System.ClientModel/samples/ServiceMethods.md#protocol-methods
from here: https://github.com/Azure/autorest.csharp/issues/4481#issuecomment-2021678802

Reply:
It should be https://aka.ms/azsdk/net/protocol-methods

Reply:
Sure I will change the link to this new one